### ISSUE TITLE
[Issue]: View stays completely Black

### ISSUE THREAD

**Issue by DerPlayer2001** [OpenTOPAS/OpenTOPAS#85]
### Contact Details

derplayer2001@hobbylos.org

### What was the issue?

After fresh install(Win 11/WSL) when running the example `examples/Graphics/QtTest.txt` (as instructed by the Install Guide) I get the expected console output(shown below) also the UI pops up as expected: 
<img width="654" alt="Topas" src="https://github.com/user-attachments/assets/32837c2a-ffcf-4b06-a9f9-164a2b91796a">  
but as seen in the picture ViewA stays completely black. This persists with other parameter files changing the view type didn't effect the result. Thanks in advance for looking into this.

### Version

4.0

### What Operative System (OS) are you running on?

Windows (WSL)

### Relevant log output

```shell
Welcome to TOPAS, Tool for Particle Simulation (Version 4.0)
Loading parameters starting from: /home/user/Applications/TOPAS/OpenTOPAS-install/examples/Graphics/QtTest.txt

Geant4 Data directory has been specified by the
TOPAS_G4_DATA_DIR environment variable as /home/user/Applications/GEANT4/G4DATA

**************************************************************
 Geant4 version Name: geant4-11-01-patch-03 [MT]   (10-November-2023)
  << in Multi-threaded mode >> 
                       Copyright : Geant4 Collaboration
                      References : NIM A 506 (2003), 250-303
                                 : IEEE-TNS 53 (2006), 270-278
                                 : NIM A 835 (2016), 186-225
                             WWW : http://geant4.org/
**************************************************************

Available UI session types: [ Qt, tcsh, csh ]
MESA: error: ZINK: failed to choose pdev
glx: failed to create drisw screen
TOPAS is in MT mode, setting number of threads to: 1

TOPAS set the tolerances based on the World size to:
      Surface tolerance = 0.05 nm, and radial tolerance = 0.05 nm.
Checking overlaps for volume MyBox:0 (G4Box) ... OK! 

Checking for Unused Components.
These are Components that were never built even though they were at least
partially defined in some parameters. This is not necessarily an error.
It may be an intentional choice to have these Component definitions handy for later use.
To turn off this check, set Ge/CheckForUnusedComponents to False
  No unused components were found.

This command is no longer relevant. There is no longer any limit on the
number of display lists - except, of course, the available memory in
your computer. Keep an eye on that. Good luck!
Pausing at Geant4 command line before starting run sequence.
Use Qt widgets to adjust TOPAS Parameters, Components, Scorers, etc.
Then hit the right arrow on the right side of the main menu to execute the TOPAS Run Sequence.
You can also enter Geant4 commands on this command line, but note that changes made
this way are outside of TOPAS control and can result in unreliable behavior.
 Changing export format to "jpg"
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.
